enarx-keep-sgx-shim/.cargo/config

```diff
@@ -2,6 +2,5 @@
 target = "x86_64-unknown-linux-musl"
 rustflags = [
     "-C", "relocation-model=pic",
-    "-C", "opt-level=3",
```

Comment:
Why don't we want this? Is this automatically included in the release profile?

Reply:
Yes, this is automatic in `release`... and we don't want this in `debug`

Reply:
e.g. it removes all `debug_assert` in debug mode

Reply:
ACK